installer_desktopselection-has-server-selection-text-20170317.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "type": "match",
+      "width": 167,
+      "xpos": 52,
+      "ypos": 367,
+      "height": 16
+    }
+  ],
+  "tags": [
+    "has-server-selection"

```

Comment:
I am not quite sure if maybe at least this screen is already influenced by https://github.com/yast/yast-installation/pull/537